### ISSUE TITLE
Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly 'CrystalDecisions.ReportAppServer.CubeDefModel,

### ISSUE THREAD

**Issue by sunk818** [ajryan/RptToXml#31]
c:\> rpttoxml "c:\docs\report.rpt" c:\temp\44.xml
Dumping c:\docs\report.rpt

Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly 'CrystalDecisions.ReportAppServer.CubeDefModel, Version=13.0.2000.0, Culture=neutral, PublicKeyToken=692fbea5521e1304' or one of its d
ependencies. The system cannot find the file specified.
   at RptToXml.RptDefinitionWriter..ctor(String filename)
   at RptToXml.Program.Main(String[] args) in c:\Development\RptToXml\RptToXml\Program.cs:line 57

**Comment by sunk818** [comment]
You have to download the Crystal Run Time:

https://blogs.sap.com/2016/12/06/sap-crystal-reports-developer-version-for-microsoft-visual-studio-updates-runtime-downloads/

**Comment by gonzalodeniz** [comment]
It works with "CRRuntime_32bit_13_0_20.msi"

**Comment by sunk818** [comment]
@gonzalodeniz I am using Windows 10 64-bit but I still had to install the 32-bit version and needed this 13.0 version. The newer downloads from SAP error out. Since this specific version is vital, it seems we have to depend on it unless the code is updated to support newer versions of CRforVS? 

Anyway, I saved a copy of the zip file in my repo here:
https://github.com/sunk818/RptToXml/blob/master/RptToXml/CRforVS/CRforVS_redist_install_32bit_13_0_20.zip

I tried to find it on SAP but they don't go that far back, requires registration possibly, etc.
